setup.py

```diff
@@ -1,7 +1,6 @@
 # SPDX-License-Identifier: GPL-3.0+
 from setuptools import setup, find_packages
 
-requirements = []
```

Comment:
This change is fine but it's not necessary and unrelated to the PR. Could you please readd this line?

Reply:
I had a mypy error due this line. https://travis-ci.org/github/release-engineering/estuary-api/builds/702283638
So I should delete this line or add a type definition here. So I've decided to delete  it.